packages/components/Notification/src/NotificationTokens.ts

```diff
@@ -4,41 +4,61 @@ import { NotificationTokens } from './Notification.types';
 
 export const defaultNotificationTokens: TokenSettings<NotificationTokens, Theme> = (t: Theme) =>
   ({
+    alignSelf: 'flex-start',
     backgroundColor: t.colors.background,
+    borderColor: 'transparent',
+    borderRadius: 12,
+    borderWidth: 1,
     color: t.colors.brandForeground1,
     fontFamily: 'primary',
+    fontLetterSpacing: -0.24,
```

Comment:
Because this is iOS specific, it might be good to mark so with a comment " //iOS only prop" at the end. That way, if/when we bring Notification to other platforms like android, it's easier for us to know what to move around